fenix/app/src/main/res/xml/search_settings_preferences.xml

```diff
@@ -72,5 +72,17 @@
             android:layout="@layout/checkbox_left_preference"
             android:title="@string/preferences_show_search_suggestions_in_private"
             app:iconSpaceReserved="false" />
+        <SwitchPreference
+            app:iconSpaceReserved="false"
+            android:defaultValue="false"
```

Comment:
Will the default value for these be true once this is shipped to release? If so it would be nice for that to be captured here, but that could be done in a follow-up

Reply:
Yep, it will! Do you think I should make it `true` now, or remove the `android:defaultValue` attribute entirely? The fragment will still update `isVisible` and `isChecked` depending on the feature flag, so I'm wondering if I need this at all...

Reply:
Short answer: you probably don't need it at all.

Longer: The `isChecked` property is updated to the real value during `onResume` in `SearchEngineFragment`. These XML preferences will technically be rendered prior, during `onCreateView`. The re-render shouldn't ever be noticeably on even the slowest devices though. It does look like there is a [method](https://developer.android.com/reference/androidx/preference/PreferenceManager#setDefaultValues(android.content.Context,int,boolean)) of ingesting these values to control disk-backed values, but AFAICT we aren't actually using them here since we have our own bespoke `Settings` utility.

Reply:
Great; I'll go ahead and remove the attributes, then, and test it out to make sure that flipping the secret setting and opening Search settings shows these new switches as enabled.